### DISCUSSION TITLE
Explore: option to start with logs + 15min timeframe

### DISCUSSION THREAD

**Discussion by BitProcessor** [grafana/grafana#37465] [Dashboards, Panels & Visualizations]
I think it might be a good idea to make the "Explore" window a little bit more configurable when it comes to the defaults when opening the window.

I think most people (can be very wrong) use this window to explore logs?
If that's the case, it's pretty annoying that it opens Elasticsearch (my default datasource + logs datasource) in "count" mode with a default "Last 1 hour" interval.

I would like to see it possible to default to "logs" (instead of "count") and "Last 15min".
Anything planned like that to make this configurable?

Or maybe I'm missing an option somewhere and this is already possible?

**Comment by gabor** [comment]
as these two features (explore-time-range and elastic-mode-in-explore) refer to different areas in grafana, i created a separate discussion for the elastic-metric-change at https://github.com/grafana/grafana/discussions/61856 , so this discussion can focus on the configurable explore-default-time-range.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
